tree/ntuple/v7/src/RNTupleDescriptor.cxx

```diff
@@ -32,8 +32,9 @@
 #include <set>
 #include <utility>
 
+using namespace ROOT::Experimental;
```

Comment:
I put it there to reduce the type noise in the file (though arguably it might belong to a separate PR).
I would advocate for abbreviating away the namespace that a cxx file belongs in most cases, either through `using namespace` or by surrounding everything in `namespace ROOT::Experimental {}`, but yes, it's a matter that should be discussed and agreed upon.

Reply:
Right, but for the current PR it seems to increase the diff for this file from <100 added lines at the end to an additional 150 changed lines because of type simplifications...

Reply:
I opened https://github.com/root-project/root/pull/16193 with just the namespace change, I'll rebase this PR on that one when it gets merged